plugin/evm/message/codec.go

```diff
@@ -15,19 +15,17 @@ const (
 	maxMessageSize = 2*units.MiB - 64*units.KiB // Subtract 64 KiB from p2p network cap to leave room for encoding overhead from AvalancheGo
 )
 
-var (
-	Codec codec.Manager
-)
+var Codec codec.Manager
```

Comment:
nit unneeded diff, let's revert it

```suggestion
var (
	Codec codec.Manager
)
```